.github/workflows/test.yml

```diff
@@ -3,7 +3,9 @@ name: ovn-ci
 on:
   merge_group:
   pull_request:
-    branches: [ master ]
+    branches:
```

Comment:
@flavio-fernandes thanks for the PR! I think we also need to alter all the spots in kind script which uses @master to point to @releave-1.0 for the artifacts no?

Reply:
Iirc, the kind scripts are building the images that they use; not pulling any @master . 
You may be thinking of the build-master job? I mean this one:  https://github.com/ovn-org/ovn-kubernetes/blob/master/.github/workflows/test.yml#L58

I see that is only used for the ovn-upgrade-e2e job:    https://github.com/ovn-org/ovn-kubernetes/blob/master/.github/workflows/test.yml#L249 . And that is okay, as it is intended that upgrades from PR to master work as expected.